### PR TITLE
feat: Add ApeChain network and token icons

### PR DESCRIPTION
**Pull request by gambinish**
## **Description**

Mobile version of: https://github.com/MetaMask/metamask-extension/pull/27974

## **Manual testing steps**

Add ApeChain custom network, with APE symbol. Native token should be token icon, network icons should be APE network icon.

## **Screenshots/Recordings**

Screenshot 2024-10-21 at 12 26 38 PM
Screenshot 2024-10-21 at 12 26 43 PM
Screenshot 2024-10-21 at 12 26 53 PM

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/util/networks/customNetworks.tsx [hunk 1]
```diff
@@ -130,7 +130,10 @@ export const UnpopularNetworkList = [
   },
 ];
 
+console.log('THIS: ', toHex(33139));
+
 export const CustomNetworkImgMapping: Record<`0x${string}`, string> = {
   '0xe': require('../../images/flare-mainnet.png'), // Flare Mainnet
   '0x13': require('../../images/songbird.png'), // Songbird Testnet
+  [toHex(33139)]: require('../../images/ape-network.png'),
 };
```
Comment (sahar-fehri) [inline app/util/networks/customNetworks.tsx:139]: Can we maybe not use the toHex to match the other mappings?
These are the chainIds for testnet and mainnet (0x8157, 0x8173)

**Comment by sahar-fehri** [comment]
Hey @gambinish , can we undraft this one? Were you planning to add something else to it?

File: app/util/networks/customNetworks.tsx [hunk 2]
```diff
@@ -130,7 +130,10 @@ export const UnpopularNetworkList = [
   },
 ];
 
+console.log('THIS: ', toHex(33139));
+
```
[the remark this answers is not in the document]
Comment (gambinish) [reply app/util/networks/customNetworks.tsx]: https://github.com/MetaMask/metamask-mobile/pull/11928/commits/581e73e12cdcae5a47abb81b62f599f167ab174b

File: app/util/networks/customNetworks.tsx (hunk 1 above)
Comment (gambinish) [reply app/util/networks/customNetworks.tsx:138]: https://github.com/MetaMask/metamask-mobile/pull/11928/commits/581e73e12cdcae5a47abb81b62f599f167ab174b

**Review by sahar-fehri — APPROVED** [review]

**Review by tommasini — APPROVED** [review]

**Pull request merged**